pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -27,6 +27,7 @@ const upStatus = 1
 
 var (
 	healthCheckInterval = 60 * time.Second
+	nodeCheckInterval   = 60 * time.Second
```

Comment:
Is this interval used somewhere? Or we just trigger it on the "node updated" event?

Reply:
This made it in accidentally from one of my earlier implementations, thanks for catching it!